Assets/Post Processing/Grayscale.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using UnityEngine;
+using UnityEngine.Rendering.PostProcessing;
+
+[Serializable]
+[PostProcess(typeof(GrayscaleRenderer), PostProcessEvent.AfterStack, "Custom/Grayscale")]
+public sealed class Grayscale : PostProcessEffectSettings
+{
+    [Range(0f, 1f), Tooltip("Grayscale effect intensity.")]
+    public FloatParameter blend = new FloatParameter { value = 0.5f };
+}
+
+public sealed class GrayscaleRenderer : PostProcessEffectRenderer<Grayscale>
+{
+    public override void Render(PostProcessRenderContext context)
+    {
+        Debug.Log(settings.blend.value);
```

Comment:
maybe add some additional string to debug.log to have clear indication that value that is shown means blend value?

Reply:
oh I just have to remove the debug :D